core/shared/platform/zephyr/platform_internal.h

```diff
@@ -1,6 +1,8 @@
 /*
  * Copyright (C) 2019 Intel Corporation.  All rights reserved.
  * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ * SPDX-FileCopyrightText: 2024 Siemens AG (For Zephyr usermode changes)
```

Comment:
If using a new license, had better add the license file under current folder, and then add the related info in `<wamr_root>/ATTRIBUTIONS.md` (refer to other components and licenses), could you help do that?

Reply:
I'm currently working on this as I have to check for internal regulations. I will post any updates as soon as I know something. Thank you for pointing out the `ATTRIBUTIONS.md`